server/routes/dashboard.js

```diff
@@ -70,7 +70,7 @@ router.post('/api/search/reference', async (req, res) => {
                 return res.json({ forwardProps: { workstack: res.locals.workstack }, redirect: '/search/reference' });
             }
         }
-        res.json({ errors: { 'case-reference': 'No active workflows for case' } });
+        res.json({ errors: { 'case-reference': 'No cases have that reference - check what you have entered and try again' } });
```

Comment:
The wording for this is inaccurate. You could enter a case reference of a closed case and it wouldn't come up, but if you searched for the case with the case reference via Search, Search would return the closed case.

This needs putting back to Design to refine the wording.